### PR TITLE
DEVDOCS-4719: [migrate] Legacy Products, migrate syntax

### PR DESCRIPTION
**Pull request by bc-andreadao**
# [DEVDOCS-4719]

Migration syntax for Legacy Products

[DEVDOCS-4719]: https://bigcommercecloud.atlassian.net/browse/DEVDOCS-4719?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/legacy/v2-catalog-products/v2-products.md [hunk 1]
```diff
@@ -98,7 +99,7 @@ A product object represents a saleable item in the catalog.
...
+| avalara_product_tax_code | resource | Accepts AvaTax System Tax Codes, which identify products and services that fall into special sales-tax categories. By using these codes, merchants who subscribe to Avalara Premium can calculate sales taxes more accurately.<br /><br /> Stores without Avalara Premium will ignore the code when calculating sales tax. Do not pass more than one code. The codes are case-sensitive.<br /><br /> For details, please see Avalara's <a href="https://help.avalara.com/000_Avalara_AvaTax/Manage_Product_Taxability/010_Select_AvaTax_System_Tax_Codes" target="_blank">overview</a> and <a href="https://help.avalara.com/000_Avalara_AvaTax/Manage_Product_Taxability/Tax_Codes_-_Frequently_Asked_Questions" target="_blank">FAQ</a  on AvaTax System Tax Codes. You can also <a href="https://help.avalara.com/@api/deki/files/1675/AvaTax_System_Tax_Codes_Toolkit.zip?revision=37" target="_blank">download codes</a> as a zipfile of spreadsheets, or <a href="https://taxcode.avatax.avalara.com/" target="_blank">search or browse codes</a> in Avalara's Tax Code Search Tool. (These external links are subject to change.) |
```
[the remark this answers is not in the document]
Comment (bc-andreadao) [reply docs/legacy/v2-catalog-products/v2-products.md:102]: Not sure what this is referring to (don't see an `an` or `on` in that line)

[the remark this answers is not in the document]
Comment (slsriehl) [reply docs/legacy/v2-catalog-products/v2-products.md]: this is linting the `<a ` tags as if they were sentences. no action needed.

**Review by bc-tgomez — COMMENTED** [review]
Some suggestions and changes needed.

**Review by bc-tgomez — COMMENTED** [review]
Some suggestions and changes needed.

**Review by bc-traciporter — APPROVED** [review]
All files render

**Review by slsriehl — APPROVED** [review]

**Pull request merged**